parfive/main.py

```diff
@@ -12,18 +12,24 @@ def main():
                         help='Number of maximum connections.')
     parser.add_argument('--overwrite', action='store_const', const=True, default=False,
                         help='Overwrite if the file exists.')
-    parser.add_argument('--no-file-progress', action='store_const', const=False, default=True, dest='file_progress',
-                        help='Overwrite if the file exists.')
+    parser.add_argument('--no-file-progress', action='store_const', const=True, default=False, dest='no_file_progress',
+                        help='Show progress bar for each file.')
     parser.add_argument('--directory', type=str, default='./',
                         help='Directory to which downloaded files are saved.')
+    parser.add_argument('--print-filenames', action='store_const', const=True, default=False, dest='print_filenames',
```

Comment:
I would think this would be better the other way around? Defaults to True, flag to disable?

Reply:
I'm sticking to unix philosophy. If success, don't print anything. :P. I can change if you still want though.